### DISCUSSION TITLE
Landscape+Location+Timezone all-in-one

### DISCUSSION THREAD

**Discussion by Paul-Gilmartin** [Stellarium/stellarium#3279] [Ideas]
In one landscape.ini I see:
[landscape]
name = Guereins
    ...
[location]
planet = Earth 
latitude = +46d6'31"
longitude = +4d46'49"
altitude = 83
timezone = Europe/Paris

It would be useful to set all this information by selecting a single file rather than one line at a time.
(Is there already a way of doing that?)

"Get location from network" is useless to me -- it gives the location of my VPN server.

Use system location would be valuable but most OSes protect that for security reasons.

Thanks,
gil

**Comment by Paul-Gilmartin** [comment]
Found it (RTFM)  it's 4.4.5 The Landscape Tab "Use associated planet and position".

Now I find 4.2.1 Time Zones “Enable daylight saving time”, also this civilisatory development,
"civilisatory" seems to be a mistranslation (of what?)

Appendix D lacks a description of "flag_dst" which I found in config.ini.

Thanks,
gil

**Reply by gzotti** [reply]
Best solution is to store your position and not call location from network everytime - this is a first-startup solution that works for the majority of users. We naively assume people would read the intro chapter of the User Guide...

civilisatory: I wanted to express something along the lines of what my dictionary now proposes: "lifestyle disease" or "disease of civilisation". 

Thanks for the missed flag ... fixed.
